### ISSUE TITLE
Unclear about data files

### ISSUE THREAD

**Issue by dbalaram** [rajat503/zero-shot-classification#1]
hey,
Thanks for the code. I was just wondering whether you have pretrained models

**Comment by rajat503** [comment]
Hi,
I don't have any pre-trained model, but training it doesn't take much time
on GPU. It should work pretty much straight away, given you have CIFAR
dataset in the directory.

**Comment by dbalaram** [comment]
Hey,

Thanks for the quick reply. I'm actually using this code for my project
baseline. It would be super helpful if you could tell me the order in which
to run all the files.

Thanks a lot!

Regards,
Dhanashree Balaram
Department of Electrical and Computer Engineering
Carnegie Mellon University

**Comment by rajat503** [comment]
read_data_cifar10.py starts the training and computes the accuracy, but I
haven't implemented checkpointing to save the model. You'll need to have
the GLoVe embeddings (available on the Stanford website) and the CIFAR-10
dataset in the directory.

**Comment by dbalaram** [comment]
Hey,

So I ran the code and the loss seems to increase. (fluctuate mostly). Where
do you think the error might be?

**Comment by rajat503** [comment]
A constant increase in loss seems unlikely and I don't remember this happening for me. L2 loss can have some variations initially but it eventually converges. Can you try the classification based model instead of the regression model? Just comment and  uncomment the last line in read_data_cifar10.py to switch between the two models.

**Comment by rajat503** [comment]
Although, if I remember correctly, the regression model worked better. Should have documented everything somewhere and written better quality code :(
